tests/cases/compiler/exportDefaultEnumTargetES2015ModuleAMD.ts

```diff
@@ -0,0 +1,62 @@
+// https://github.com/Microsoft/TypeScript/issues/3792

```

Comment:
Consider adding these tests in a relevant location under `tests/cases/conformance/**` as `tests/cases/compiler` is extremely cluttered and unorganized.

Reply:
Also, consider adding `// @declaration: true` to verify .d.ts emit and `// @sourceMaps: true` to verify source-map emit.